cmd/app/command/iscsi-discover.go

```diff
@@ -0,0 +1,45 @@
+package command
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/openebs/node-disk-manager/cmd/storage/iscsi"
+	"github.com/openebs/node-disk-manager/pkg/util"
+	"github.com/spf13/cobra"
+)
+
+type CmdIscsiDiscoverOptions struct {
+	target string
+}
+
+//NewSubCmdIscsiDiscover discovers iscsi block devices
+func NewSubCmdIscsiDiscover() *cobra.Command {
+	options := CmdIscsiDiscoverOptions{}
+
+	getCmd := &cobra.Command{
+		Use:   "discover",
+		Short: "discover block device with iscsi",
+		Long:  `the  block device is discovered on the storage area network with specified target`,
+		Run: func(cmd *cobra.Command, args []string) {
+
+			util.CheckErr(options.Validate(), util.Fatal)
+			res, err := iscsi.IscsiDiscover(options.target)
```

Comment:
Just wondering how the res looks like. Has it be been handled to display json or yaml like formats. Just a query. This is not a review comment :)